oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -339,6 +344,36 @@ public boolean updateAttributes() {
         return true;
     }
 
+    /**
+     * If some details couldn't be read from ProcPath.PID_STATUS try reading it from
+     * ProcPath.PID_STAT
+     *
+     * @param status
+     *            status map to fill.
+     * @param stat
+     *            string to read from.
+     */
+    private static void getMissingDetails(Map<String, String> status, String stat) {
+        if (status == null || stat == null) {
+            return;
+        }
+
+        int nameStart = stat.indexOf("(");
+        int nameEnd = stat.indexOf(")");
```

Comment:
But [benchmarks show no real difference](https://stackoverflow.com/questions/33646781/java-performance-string-indexofchar-vs-string-indexofsingle-string#:~:text=1%20Answer&text=As%20you%20can%20see%2C%20indexOfChar,but%20somewhat%20slower%20for%20indirect.) so I don't care that much.